gr-uhd/grc/gen_uhd_usrp_blocks.py

```diff
@@ -119,10 +119,10 @@
 % if sourk == 'sink':
 
 outputs:
+% endif
```

Comment:
Would the `if/endif` would be removed entirely in this case?

Reply:
You still need that `if/endif`. Otherwise there will be two `outputs` labels.
The `stream` port is an input for the sink and an output for the source.

Reply:
The UHD yml is being tricky at the expense of readability ... OK, not your problem.